### PR TITLE
Feat/m vpc

### PR DESCRIPTION
**Pull request by Abhishekism9450**
Added Resource for VPC:

- resource_nutanix_vpc

Added DataSources for VPC:

- data_source_nutanix_vpc
- data_source_nutanix_vpc_list

### CODE REVIEW

File: client/v3/v3_service.go [hunk 1]
```diff
@@ -2428,3 +2433,67 @@ func (op Operations) UpdateAddressGroup(uuid string, body *AddressGroupInput) er
 
 	return op.client.Do(ctx, req, nil)
 }
+
+func (op Operations) CreateVPC(ctx context.Context, request *VPCIntentInput) (*VPCIntentResponse, error) {
+	//ctx := context.TODO()
```
Comment (siddharth-kulshrestha) [inline client/v3/v3_service.go]: Please remove this unwanted line of code.

File: nutanix/data_source_nutanix_vpc.go [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+func dataSourceNutanixVPC() *schema.Resource {
+	return &schema.Resource{
+		ReadContext: dataSourceNutanixVPCRead,
+		Schema: map[string]*schema.Schema{
+			"vpc_uuid": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+
+			// COMPUTED RESOURCES
```
Comment (siddharth-kulshrestha) [inline nutanix/data_source_nutanix_vpc.go]: Comp
```suggestion
			// Computed attributes
```

File: nutanix/data_source_nutanix_vpc_list.go [hunk 3]
```diff
@@ -0,0 +1,391 @@
...
+				Type:     schema.TypeInt,
+				Optional: true,
+			},
+			"offset": {
+				Type:     schema.TypeInt,
+				Optional: true,
+				Default:  0,
+			},
+
+			// COMPUTED RESOURCES
```
Comment (siddharth-kulshrestha) [inline nutanix/data_source_nutanix_vpc_list.go]:
```suggestion
			// Computed attributes
```

File: nutanix/resource_nutanix_vpc.go [hunk 4]
```diff
@@ -0,0 +1,487 @@
...
+		if response.Spec.Resources != nil {
+			res = response.Spec.Resources
+		}
+	}
+
+	if d.HasChange("name") {
+		spec.Name = utils.StringPtr(d.Get("name").(string))
+	}
+
+	// if d.HasChange("external_subnet_list") {
```
Comment (siddharth-kulshrestha) [inline nutanix/resource_nutanix_vpc.go]: Is this commented code required?

**Review by siddharth-kulshrestha — CHANGES_REQUESTED** [review]

File: nutanix/resource_nutanix_vpc.go (hunk 4 above)
Comment (Abhishekism9450) [reply nutanix/resource_nutanix_vpc.go]: No. Removed the commented code.

**Review by siddharth-kulshrestha — APPROVED** [review]

**Pull request merged**